associated-token-account/program/src/instruction.rs

```diff
@@ -34,6 +34,20 @@ pub enum AssociatedTokenAccountInstruction {
     ///   4. `[]` System program
     ///   5. `[]` SPL Token program
     CreateIdempotent,
+    /// Transfers from and closes a nested associated token account: an
+    /// associated token account owned by an associated token account.
```

Comment:
how about adding like this? imo, we should be explicit about the context of this ix's existence.
`note: The nested token account is most likely created unintentionally and this instruction shouldn't be used casually.`

Reply:
I made the language even stronger than your suggestion :smile: